Formula/d/duplicity.rb

```diff
@@ -285,8 +325,8 @@ class Duplicity < Formula
   end
 
   resource "pbr" do
-    url "https://files.pythonhosted.org/packages/8d/c2/ee43b3b11bf2b40e56536183fc9f22afbb04e882720332b6276ee2454c24/pbr-6.0.0.tar.gz"
-    sha256 "d1377122a5a00e2f940ee482999518efe16d745d423a670c27773dfbc3c9a7d9"
+    url "https://files.pythonhosted.org/packages/c3/2c/63275fab26a0fd8cadafca71a3623e4d0f0ee8ed7124a5bb128853d178a7/pbr-1.10.0.tar.gz"
+    sha256 "186428c270309e6fdfe2d5ab0949ab21ae5f7dea831eab96701b86bd666af39c"
```

Comment:
this needs to be reverted

Reply:
yea a bunch of versions got downgraded, not sure why

Reply:
This is an upstream bug (I don't have a gitlab account to report myself), they are now properly declaring their dependencies but are all pinned and some to very old versions: https://gitlab.com/duplicity/duplicity/-/blob/dev/requirements.txt

Reply:
:(